platform/core/src/extensions/ExtensionManager.js

```diff
@@ -281,6 +281,7 @@ export default class ExtensionManager {
 
       return extensionModule;
     } catch (ex) {
+      console.log('Exception:', ex);
```

Comment:
what is the need for log, when there is Error below?

Reply:
The error below swallows this stack trace, so there is nothing obvious any longer.  It makes possible debugging extension load issues - and those shouldn't occur in production, so I don't see the problem with the exception.